zeebe/dynamic-config/src/main/java/io/camunda/zeebe/dynamic/config/api/ClusterConfigurationRequestTopics.java

```diff
@@ -22,6 +22,7 @@ public enum ClusterConfigurationRequestTopics {
 
   SCALE_CLUSTER("topology-cluster-scale"),
   PATCH_CLUSTER("topology-cluster-patch"),
+  PURGE("topology-purge"),
```

Comment:
:wrench: I don't really care about the topology in there (I guess we can remain consistent), but would like at least the `cluster`, to highlight this is specifically about purging the complete cluster.